src/components/Table/BlueprintTable.js

```diff
@@ -2,24 +2,71 @@ import React from "react";
 import PropTypes from "prop-types";
 import { Link } from "react-router-dom";
 import { useIntl } from "react-intl";
-import { TableComposable, Tr, Tbody, Td } from "@patternfly/react-table";
+import {
+  TableComposable,
+  Th,
+  Thead,
+  Tr,
+  Tbody,
+  Td,
+} from "@patternfly/react-table";
 import CreateImageWizard from "../Wizard/CreateImageWizard";
 import DeleteBlueprint from "../Modal/DeleteBlueprint";
 import ExportBlueprint from "../Modal/ExportBlueprint";
 
+const getLatestCreationDate = (images, blueprint) => {
+  const imagesByBlueprint = images.filter(
+    (image) => image.blueprint === blueprint.name
+  );
+  if (imagesByBlueprint.length > 0) {
+    const latestTimestamp = Math.max(
+      ...imagesByBlueprint.map((image) => image.job_finished)
+    );
+    return timestampToDate(latestTimestamp);
+  } else {
+    return "-";
+  }
+};
+
+const timestampToDate = (timestamp) => {
+  var options = { year: "numeric", month: "short", day: "numeric" };
+  var date = new Date(timestamp * 1000);
+  return date.toLocaleDateString("en-US", options);
+};
+
 const BlueprintTable = (props) => {
   const intl = useIntl();
 
   return (
     <TableComposable
       aria-label={intl.formatMessage({ defaultMessage: "Blueprints table" })}
+      variant="compact"
     >
+      <Thead>
+        <Tr>
+          <Th>Name</Th>
+          <Th>Version</Th>
+          <Th>Last image created</Th>
+          <Th>Images built</Th>
+          <Th>Packages</Th>
+        </Tr>
+      </Thead>
       <Tbody>
         {props.blueprints.map((blueprint) => (
           <Tr key={blueprint.name} data-testid={blueprint.name}>
             <Td>
               <Link to={`/${blueprint.name}`}>{blueprint.name}</Link>
             </Td>
+            <Td>{blueprint.version}</Td>
+            <Td>{getLatestCreationDate(props.images, blueprint)}</Td>
+            <Td>
+              {
+                props.images.filter(
+                  (image) => image.blueprint === blueprint.name
+                ).length
+              }
```

Comment:
Can you move this to a seperate function like `getLatestCreationDate`

Reply:
Moved :+1: